src/ontology/mondo-edit.obo

```diff
@@ -410932,7 +410932,8 @@ id: MONDO:0027749
 name: serpinopathy
 subset: ordo_group_of_disorders
 xref: Orphanet:250805 {source="MONDO:equivalentTo"}
-intersection_of: MONDO:0000001 ! disease or disorder
+is_a: MONDO:0003847 {source="https://orcid.org/0000-0002-4142-7153"} ! Mendelian disease
```

Comment:
you can add Orphanet as a source too, because Orphanet classifies this as a Rare genetic disease, which is basically the same as our Mendelian disease, see:
https://www.ebi.ac.uk/ols/ontologies/ordo/terms?iri=http%3A%2F%2Fwww.orpha.net%2FORDO%2FOrphanet_250805

add source: Orphanet:250805